dev/io.openliberty.microprofile.openapi.2.0.internal/src/io/openliberty/microprofile/openapi20/cache/CacheEntry.java

```diff
@@ -240,42 +253,49 @@ public void write() throws IOException {
                 if (fileEntries.isEmpty())
                     Tr.debug(this, tc, "Not writing cache entry, fileEntries is empty");
```

Comment:
I don't see that CacheEntry redefines toString in order to get the appName, model etc in the Tr.debug s ?

Reply:
It doesn't. `this` as the first argument is only used to provide the value for the `id=xxxxxxxxx` column of the trace log to identify the instance that logged the trace message.

We're not tracing out the app name or model here as we will have done so before, so in context of the surrounding trace messages it should be clear enough which app and model is being processed.

Reply:
OK